awslabs/fhir-works-on-aws-deployment
# feat: Group export scripts

Issue #, if available:

Description of changes:
* Add script to filter out search parameters needed by patient compartment 
* Update script and serverless.yml to upload the generated JSON file to S3 for Glue job to reference
* Tested JSON files are uploaded successfully and can be used by the Glue job 
* Update export script to handle group export
* Manually tested group export E2E 
* WIP - integ tests 

Checklist:




By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
